easybuild/easyconfigs/r/rocm-smi/rocm-smi-5.4.4-GCCcore-11.3.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'rocm-smi'
+version = '5.4.4'
+
+homepage = 'https://github.com/RadeonOpenCompute/rocm_smi_lib'
+description = """The ROCm System Management Interface Library, or ROCm SMI
+library, is part of the Radeon Open Compute ROCm software stack. It is a C
+library for Linux that provides a user space interface for applications to
+monitor and control GPU applications."""
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ["https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/"]
+sources = ["rocm-%(version)s.tar.gz"]
+checksums = ['27a6d851a493a3ab11ca3af079b4a0509579aa7bab2b2891af3bdf816a456ff0']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/rocm-smi'],
+    'dirs': [''],
```

```suggestion
    'dirs': [],
```
if there aren't any important dirs to look for then list should just be empty.